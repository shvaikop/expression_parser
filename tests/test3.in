7
15.567 * 4 - 3
20 / 3
20.0 / 3
21 / 4 + -20.0 / 6
31.5 % 2
123 + 5 * -123.4 -     5
60    * 4 % 3 + 21.5/ 1.5 
