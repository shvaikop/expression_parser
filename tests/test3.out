59.268
6
6.66667
1.66667
invalid
-499
14.3333
